Controllers/HomeController.cs

```diff
@@ -9,11 +10,13 @@ public class HomeController : Controller
     {
         private readonly DbContextOptions<DataBaseContext> _options;
         private readonly IHtmlLocalizer<HomeController> _localizer;
+        private readonly ILogger<HomeController> _log;
 
-        public HomeController(DbContextOptions<DataBaseContext> options, IHtmlLocalizer<HomeController> localizer)
+        public HomeController(DbContextOptions<DataBaseContext> options, IHtmlLocalizer<HomeController> localizer, ILogger<HomeController> log)
         {
             this._options = options;
             this._localizer = localizer;
+            this._log = log;

```

Comment:
is necesary this attribute? maybe with the config in the Startup It's enough

Reply:
yes, it is necessary in each controller